examples/gnrc_networking_mac/mac.c

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2018 SHUGUO ZHUO
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     examples
+ * @{
+ *
+ * @file
+ * @brief       MAC shell commands
+ *
+ * @author      Shuguo Zhuo <zhuosgzju@gmail.com>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <inttypes.h>
+
+#include "net/gnrc.h"
+#include "net/gnrc/mac/types.h"
+
+int mac_cmd(int argc, char **argv)
+{
+    if (argc < 2) {
+        printf("usage: %s duty\n", argv[0]);
+        return 1;
+    }
+
+    if (strcmp(argv[1], "duty") == 0) {
+
+        gnrc_netif_t *netif = NULL;
+        netif = gnrc_netif_iter(netif);
+
+        msg_t msg;
+        msg.type = GNRC_MAC_TYPE_GET_DUTYCYCLE;
+        msg_send(&msg, netif->pid);
+    }
+    else {
+        puts("error: invalid command");
```

Comment:
if there is only one command, why not cal l the command `mac_duty` and don't expect an argument?

Reply:
... em.. I believe in the future, there will be more MAC commands, and this PR just introduces the first one.

Reply:
Ok.

Reply:
:smiley: